data/services/meson.build

```diff
@@ -0,0 +1,13 @@
+service_files = [
+  'org.Cinnamon.Melange.service',
+  'org.Cinnamon.Slideshow.service',
+]
```

Comment:
Should org.cinnamon.CalendarServer.service be included also?

Reply:
I moved all dbus service files into the same folder (including the $\text{org.\textbf{\textcolor{red}{c}}innamon.CalendarServer.service}$ and the $\text{org.\textbf{\textcolor{red}{C}}innamon.HotplugSniffer.service}$) in 4c3b83e4532ef21f5b265ed30f8b8cac2f4d4603, PTAL.